### ISSUE TITLE
TopTools_ListOfShape' object is not iterable

### ISSUE THREAD

**Issue by Nadjei74** [tpaviot/pythonocc-demos#75]
I can't find "TopTools_ListIteratorOfListOfShape" in from OCC.Core.TopTools

**Comment by tpaviot** [comment]
it is actually wrapped in current master branch, does it miss fro previous release?

**Comment by Nadjei74** [comment]
Yes. it is not in the previous release. So how do I access it? Do I have to update my current version of PythonOCC?
